lib/net.js

```diff
@@ -1326,84 +1326,71 @@ function listen(self, address, port, addressType, backlog, fd, exclusive) {
 
 
 Server.prototype.listen = function() {
-  var self = this;
+  const argsLen = arguments.length;
```

Comment:
This line is unnecessary, just use `arguments.length` in both places below. V8 is smart enough to optimize the `for (...; i < array.length; ...)` case.